models/Page.js

```diff
@@ -8,6 +8,8 @@ const pageSchema = new mongoose.Schema({
   content: String,
   author: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
   isPublished: Boolean,
+  metaTitle: String,
```

Comment:
Let's add a default empty string value to both `metaTitle` and `metaDescription` so that when didn't add it initially it will still have a valid value instead of `undefined`.